nautobot/dcim/api/serializers.py

```diff
@@ -768,7 +768,6 @@ class DeviceRedundancyGroupSerializer(TaggedModelSerializerMixin, NautobotModelS
     failover_strategy = ChoiceField(
         choices=DeviceRedundancyGroupFailoverStrategyChoices,
         allow_blank=True,
-        default=DeviceRedundancyGroupFailoverStrategyChoices.FAILOVER_UNSPECIFIED,
```

Comment:
Does this change the default behavior when creating a DeviceRedundancyGroup and omitting this field?

Reply:
It doesn't anymore :)

Reply:
Now it's setting it to `null` which isn't correct either. I wonder if we need to fix this in the openapi schema itself with `@extend_schema`?

Reply:
I've finally figured out the basic root cause of the original issue and it's our implementation of the `to_representation` on our `nautobot.core.api.fields.ChoiceField` that returns an OrderedDict instead of a string value. Here's where the default value is introspected in drf_spectacular: 

https://github.com/tfranzel/drf-spectacular/blob/659b1f0d00bd0bbc711cbad094226097ab3260e0/drf_spectacular/openapi.py#L1026